data/i18n/pl/pl.toml

```
@@ -130,14 +156,8 @@ other = """The Linux Foundation &reg;. All rights reserved. The Linux Foundation
 [main_documentation_license]
 other = """The Kubernetes Authors | Documentation Distributed under <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a>"""
 
-[main_edit_this_page]
-other = "Edytuj stronę"
-
-[main_github_create_an_issue]
-other = "Zgłoś problem"
-
 [main_github_invite]
-other = "Chcesz zacząć współtworzyć kod Kubernetesa?"
+other = "Chcesz zacząć mieszać w kodzie Kubernetesa?"
```

```suggestion
other = "Chcesz zacząć zabawę z kodem Kubernetesa?"
```
może tak? mniej dosłownie trochę